### ISSUE TITLE
OAS 3.0: Odd animation when expanding callbacks

### ISSUE THREAD

**Issue by hkosova** [swagger-api/swagger-ui#3450]
Version: `ft/oas3` branch

The first time the "Callbacks" section is expanded or collapsed, the animation behaves oddly.

![callback-animation](https://user-images.githubusercontent.com/8576823/28530045-1a10e46a-709b-11e7-94cb-7e3aaf19b964.gif)

Steps to reproduce:
1. Load the spec below into the UI.
2. Expand the POST.
3. Switch to the Callbacks tab.
4. Expand the callback.
5. Collapse the callback.

```
openapi: 3.0.0
info:
  title: A simple webhook subscription
  version: 1.0.0
paths:
  /subscribe:
    post:
      requestBody:
        content:
          application/json:
            schema:
              type: object
              properties:
                url:
                  type: string
                  format: uri
      responses:
        '201':
          description: Created subscription to webhook
      callbacks:
        mainHook:
          '$request.body#/url':
            post:
              responses:
                '200':
                  description: webhook successfully processed
```

**Comment by shockey** [comment]
@Minasokoni, any thoughts on this?

**Comment by shockey** [comment]
Fixed via #3522
